### PR TITLE
chore: explicitly declare nullability using JSpecify

### PR DESCRIPTION
**Pull request by TheMrMilchmann**

### CODE REVIEW

File: src/main/java/com/gw2auth/oauth2/server/service/gw2account/apitoken/Gw2AccountApiTokenServiceImpl.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ public void updateApiTokensValid(Instant lastValidCheckTime, Collection<Gw2Accou
         this.gw2AccountApiTokenRepository.updateApiTokensValid(lastValidCheckTime, updates);
     }
 
-    @Scheduled(fixedRate = 10L, timeUnit = TimeUnit.MINUTES)
+//    @Scheduled(fixedRate = 10L, timeUnit = TimeUnit.MINUTES)
```
Comment (its-felix) [inline src/main/java/com/gw2auth/oauth2/server/service/gw2account/apitoken/Gw2AccountApiTokenServiceImpl.java:68]: this should not be removed

File: src/main/java/com/gw2auth/oauth2/server/service/user/Gw2AuthUserV2.java [hunk 2]
```diff
@@ -24,16 +25,16 @@ public class Gw2AuthUserV2 implements OAuth2User, Principal, AuthenticatedPrinci
...
     private final UUID accountId;
     private final Pair<String, String> accountFederation;
-    private final String sessionId;
-    private final SessionMetadata sessionMetadata;
-    private final Instant sessionCreationTime;
-    private final byte[] encryptionKey;
+    private final @Nullable String sessionId;
+    private final @Nullable SessionMetadata sessionMetadata;
+    private final @Nullable Instant sessionCreationTime;
+    private final byte @Nullable [] encryptionKey;
```
Comment (its-felix) [inline src/main/java/com/gw2auth/oauth2/server/service/user/Gw2AuthUserV2.java:31]: can the nullable be moved to the left hand side here?

**Review by its-felix — CHANGES_REQUESTED** [review]

File: src/main/java/com/gw2auth/oauth2/server/service/user/Gw2AuthUserV2.java (hunk 2 above)
Comment (TheMrMilchmann) [reply src/main/java/com/gw2auth/oauth2/server/service/user/Gw2AuthUserV2.java:31]: Paraphrasing from what I told you elsewhere:
- `@Nullable byte[]` would be `Array<Byte?>` in Kotlin,
- whereas `byte @Nullable []` would be `Array<Byte>?`

File: src/main/java/com/gw2auth/oauth2/server/service/gw2account/apitoken/Gw2AccountApiTokenServiceImpl.java (hunk 1 above)
Comment (TheMrMilchmann) [reply src/main/java/com/gw2auth/oauth2/server/service/gw2account/apitoken/Gw2AccountApiTokenServiceImpl.java]: Good that you spotted this. I missed this after doing some testing.

**Comment by TheMrMilchmann** [comment]
Rebased and uncommented the cronjob

**Pull request merged**